akka-docs/src/main/paradox/typed/from-classic.md

```diff
@@ -90,8 +90,8 @@ Java
 
 Why is it called `Behavior` and not `Actor`?
 
-In Typed the `Behavior` defines how to handle incoming messages and after processing each message it may return
-a different `Behavior` to be used for the next message. This means that an actor is started with an initial `Behavior`
+In Typed, the `Behavior` defines how to handle incoming messages and after processing a message, a potentially
+different `Behavior` is returned for processing the next message. This means that an actor is started with an initial `Behavior`
```

Comment:
I feel this changes the meaning somewhat, I'd want to change the `each` -> `a` but partially keep the next part: `may return a different behavior for processing the next message`

Reply:
@johanandren 

Would this be ok:

_In Typed, the `Behavior` defines how to handle incoming messages. After processing a message, a different `Behavior` may be returned for processing the next message._

Reply:
@johanandren Changed.